main.tf

```
@@ -0,0 +1,115 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 4.16"
+    }
+  }
+
+  required_version = ">= 1.2.0"
+}
+
+provider "aws" {
+  region = "us-east-2"
+}
+
+resource "aws_vpc" "Main" {
+  cidr_block       = "10.0.0.0/24"
+  instance_tenancy = "default"
+  tags = {
```

Please separate the tags block from the rest of the arguments using a newline.
```suggestion
  instance_tenancy = "default"
  
  tags = {
```
Please fix this here and in all other similar places in the code